trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require '.\decorator'
```

Comment:
- [ ] It would be better if you use `require_relative './decorator'` to prevent any error :+1:

Reply:
Once again, it would be good if you please kindly at least bother to clone my repo and run it, you will come to know that it does not give error if you use backslash with require